de-lib/src/main/java/org/iplantc/de/apps/client/presenter/toolBar/AppsToolbarPresenterImpl.java

```diff
@@ -157,4 +166,14 @@ public void onSuccess(AppSharingDialog result) {
             }
         });
     }
+
+    @Override
+    public void onManageToolsClicked(ManageToolsClickedEvent event) {
+        Window w = new Window();
```

Comment:
This is the first of many PRs, so you might have changed this later, but this is probably a window we want launched through `DesktopWindowManager` with a `WindowType`  and/or `WindowConfig` like the other major windows we have.